packages/dd-trace/src/proxy.js

```diff
@@ -179,9 +179,9 @@ class Tracer extends NoopProxy {
         this._modules.appsec.enable(config)
       }
       if (!this._tracingInitialized) {
-        this._tracer = new DatadogTracer(config)
+        const prioritySampler = appsecStandalone.configure(config)
+        this._tracer = new DatadogTracer(config, prioritySampler)
```

Comment:
wdyt about this ? it removes the need for changes in the constructors and it shows the intention better i feel like, we're injecting our own sampler only if it exists.
```suggestion
        this._tracer = new DatadogTracer(config)
        const prioritySampler = appsecStandalone.configure(config)
        if (prioritySampler) {
            this._tracer._prioritySampler = priortySampler
        }
        
```

Reply:
`DatadogTracer` is creating an instance of an `Exporter` and `SpanProcessor` [here](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/opentracing/tracer.js#L32) which in turn depend on the prioritySampler instance.

We should update those references too when doing `this._tracer._prioritySampler = prioritySampler`

Reply:
Ah, alright i see, then it's not as nice, let's keep it like that